server/src/games/BGS.Games.StoneAge/StoneAgeScoringStrategy.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Text.Json;
+using BGS.ApplicationCore.Games.Interfaces;
+
+namespace BGS.Games.StoneAge;
+
+internal class StoneAgeScoringStrategy : IGameScoringStrategy
+{
+    public int Calculate(JsonElement scoringData)
+    {
+        var model = scoringData.Deserialize<StoneAgeScoringModel>();
```

Comment:
Can this value be null?

Reply:
not yet, but in the future, when we have a frontend (client), we will return to the issue of validations